var/spack/repos/builtin/packages/mstk/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mstk(CMakePackage):
+    """MSTK is a mesh framework that allows users to represent,
+    manipulate and query unstructured 3D arbitrary topology meshes
+    in a general manner without the need to code their own data
+    structures. MSTK is a flexible framework in that it allows a
+    variety of underlying representations for the mesh while
+    maintaining a common interface. It allows users to choose from
+    different mesh representations either at initialization (implemented)
+    or during the program execution (not yet implemented) so that the
+    optimal data structures are used for the particular algorithm.
+    The interaction of users and applications with MSTK is through a
+    functional interface that acts as though the mesh always contains
+    vertices, edges, faces and regions and maintains connectivity between
+    all these entities."""
+
+    homepage = "https://github.com/MeshToolkit/MSTK"
+    git      = "https://github.com/MeshToolkit/MSTK"
+
+    version("develop", branch='master', submodules=False, preferred=True)
```

Comment:
```suggestion
    version('master', branch='master')
```
Now that #1983 has been merged, you can call this version `master`. Also, submodules already defaults to False, and `preferred=True` is meaningless if you only have one version.